### ISSUE TITLE
Error squiggles being drawn over words

### ISSUE THREAD

**Issue by spaceships** [Monokai/monokai-pro-vscode#402]
Ever since VSCode 1.88.0 on Mac, the error squiggles for Monokai Pro themes have been drawn in this funny way on top of other words. Other themes draw the squiggles correctly. Haven't tried other OSes. Reverting to VSCode 1.87 fixes it for me.

<img width="205" alt="Screenshot 2024-04-08 at 11 53 09 AM" src="https://github.com/Monokai/monokai-pro-vscode/assets/1577541/6b008a55-03d2-496c-b7dc-d3d7e3b3d7ce">

**Comment by karolis-kimtys** [comment]
Same issue here.
![image](https://github.com/Monokai/monokai-pro-vscode/assets/46462840/f00ef4df-92b2-4b80-86cb-4007a25d43cb)
Using VSCode for Linux

Monokai Pro Version: 1.2.2

**Comment by fredericoo** [comment]
it’s driving me insane, took me ages to find out what the issue was

**Comment by Parthvsquare** [comment]
same issue here: 
Monokai Pro Version: 1.2.2
VSCode Version: 1.88.0
OS: Macos 14.4.1 (23E224)
<img width="281" alt="image" src="https://github.com/Monokai/monokai-pro-vscode/assets/26404957/cb0fdc9c-4ed4-4f83-84f8-b0e8c7a7ee81">

**Comment by PajariPy** [comment]
Same thing for me
![image](https://github.com/Monokai/monokai-pro-vscode/assets/25295946/c2b18555-ff5f-4589-89a3-ac831a9e1f51)
Monokai Pro Version: 1.2.2
VSCode Version: 1.88.0
OS: Ubuntu 22.04.4 LTS

**Comment by Monokai** [comment]
Seems more themes are affected by this. I'm not sure what I can do on my end. I think it's best to report over here: https://github.com/microsoft/vscode/issues/209614

**Comment by palashmon** [comment]
This seems to happening after the recent VSCode Version: 1.88.0 update. For now, you can try these temporary solutions until a fix is released by the VS Code team. https://github.com/wesbos/cobalt2-vscode/issues/248#issuecomment-2042204084

**Comment by fredericoo** [comment]
Quick solution until they release a fix:

- add the following to your user configuration:
```json
"[Monokai Pro (Filter Spectrum)]": {
      "editorError.border": "#E5484D",
      "editorWarning.border": "#FFC53D",
      "editorInfo.border": "#12A594",
      "editorError.foreground": "#00000000",
      "editorWarning.foreground": "#00000000",
      "editorInfo.foreground": "#00000000"
    }
```
- this will essentially **disable squiggly lines** and display a double border underneath the errors/warnings/info instead:
![CleanShot 2024-04-10 at 14 40 19@2x](https://github.com/Monokai/monokai-pro-vscode/assets/45042736/bbd9f049-9d14-4b43-bc89-dd1e90ae4138)

**Comment by thirtytwobits** [comment]
This is the workaround that I'm using until the bug is fixed:

```json
  "[Monokai Pro]": {
    "editorError.border":"",
    "editorWarning.border": "",
    "editorInfo.border": "",
    "editorError.foreground": ""
    "editorWarning.foreground": "",
    "editorInfo.foreground": ""
  }
```

**Comment by palashmon** [comment]
**Update:** The VS Code issue is currently tagged for the next stable release by the VS Code team which will drop in early May. Should be out in Insiders by now.

**Comment by karolis-kimtys** [comment]
This has been fixed with VSCode version 1.89.0.
![image](https://github.com/Monokai/monokai-pro-vscode/assets/46462840/f0e31042-34a8-4a7d-9102-8d3fab59c2aa)
